18F/identity-idp
# Enforce YAML normalization for application.yml.default

## 🛠 Summary of changes

Updates YAML normalization enforcement to include `application.yml.default` in enforcement.

#10981 enhanced the normalization step to _include_ `application.yml.default`, but it wasn't enforced because the pattern that looked for changes after normalization assumed that files would end in `.yml`. Since this file ends in `.yml.default`, it wasn't being included in the pattern.

## 📜 Testing Plan

1. Checkout the main branch: `git checkout main`
2. Checkout the changes to `Makefile`: `git checkout aduth-enforce-app-config-default-normalize -- Makefile`
3. Run `lint make_yaml`
4. Observe the task fails with "Error: Run 'make normalize_yaml' to normalize YAML"
